### PR TITLE
Capacity update Europe

### PR DESCRIPTION
**Pull request by KabelWlan**
Since Entsoe updated their capacity statistics, it is time for a big Europe capacity update (like I already did last year).

Besides the Entsoe updates I did the following changes as well:
wind power capacity in Norway
wind and hydro capacity in Sweden
all the capacity from Switzerland, Turkey, Ukraine and Spain

Where needed I updated the sources as well but most of them are still the same. I hope that's fine :)

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -5848,12 +5850,15 @@
       ]
     ],
     "capacity": {
-      "biomass": 151,
-      "hydro": 4810,
-      "hydro storage": 1487,
+      "biomass": 230,
+      "coal": 17242,
+      "gas": 4600
```
Comment (IV1T3) [inline config/zones.json:5855]: , missing

File: config/zones.json [hunk 2]
```diff
@@ -5848,12 +5850,15 @@
...
+      "hydro": 4829,
+      "hydro storage": 1488,
       "nuclear": 13835,
-      "solar": 4231,
-      "wind": 1036
+      "oil": 756.6
```
Comment (IV1T3) [inline config/zones.json:5859]: , missing

File: config/zones.json (hunk 1 above)
Comment (KabelWlan) [reply config/zones.json]: I was just about to look out for them but thank you, makes it a lot quicker for me.

**Review by SleepyMorpheus — COMMENTED** [review]
Please use more descriptive titles as your commit messages instead of commas... (e94d9ef3a3bbb2925a7ace967633a8aca5eb93a9)

**Comment by madsnedergaard** [comment]
> Please use more descriptive titles as your commit messages instead of commas... ([e94d9ef](https://github.com/electricitymap/electricitymap-contrib/commit/e94d9ef3a3bbb2925a7ace967633a8aca5eb93a9))

For this specific commit where he's adding commas, I think the name is fine ;) 
Also, commits are squashed when merged to `master`, so the PR title will be used for commit there.

**Review by madsnedergaard — APPROVED** [review]
Thanks for updating all of these! 💪

**Pull request merged**